sdk/purview/ci.yml

```diff
@@ -34,22 +34,26 @@ pr:
       - sdk/purview/azure-resourcemanager-purview/pom.xml
```

Comment:
please add your pom to the 2 "exclude" above.

Reply:
Fixed, thanks.